lib/_http_client.js

```diff
@@ -182,7 +182,7 @@ function ClientRequest(options, cb) {
                       'client');
     }
     self._storeHeader(self.method + ' ' + self.path + ' HTTP/1.1\r\n',
-                      self._headers);
+                      self._headersStore);
```

Comment:
Does accessing a value via a Symbol have a perf hit vs. normal property access in master?